### DISCUSSION TITLE
MAUI APK cannot install in Android device.

### DISCUSSION THREAD

**Discussion by SivaramanSivamalai** [dotnet/maui#10779] [Q&A]
I have generated a APK using default Hello  world program using .NET MAUI project. While try to install the APK in my mobile it says App not Installed. Am not sure why am getting this error. Need Help.

**Comment by acaliaro** [comment]
I think you can take a look to logcat to find some info

**Reply by SivaramanSivamalai** [reply]
Thank you. Will try it and update.

**Reply by Jay-G-HK** [reply]
Hi have exactly the same problem. Did you manage to fix it? I have published it with the correct minimum android version for my phone, but it's not working.

**Reply by achobanov** [reply]
Same here. Can you please provide more info on logcat?
